package.json

```diff
@@ -29,5 +29,10 @@
     "lerna": "^3.20.2",
     "prettier": "^2.0.5",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "@transmute/did-key-ed25519": "^0.2.1-unstable.24",
+    "@transmute/did-key-x25519": "^0.2.1-unstable.24",
+    "@w3c-ccg/universal-wallet": "file:packages/universal-wallet"
```

Comment:
this is not necessary, you can link packages of the same version with symlink using `lerna link`

Reply:
not a bad idea - will update to lerna